### PR TITLE
add terminologies

### PR DESCRIPTION
**Pull request by yuriko627**

### CODE REVIEW

File: articles/OVM_architecture.md [hunk 1]
```diff
@@ -27,26 +27,33 @@ Table of contents:
...
 [Data - Blood]
-- Claim 
-- Statement 
-- Formula 
-- Property  
-- Proof / Local info 
-    - Witness 
-    - Implication Proof 
+sometimes called as local information
+- Claim: a statement submitted to Universal Decision Contract
```
Comment (syuhei176) [inline articles/OVM_architecture.md]: Not only for UDC(and name is now UniversalAdjudicator.sol), but also client can decide the claim is whether true or false by local information.

File: articles/OVM_architecture.md [hunk 2]
```diff
@@ -27,26 +27,33 @@ Table of contents:
...
-- Property  
-- Proof / Local info 
-    - Witness 
-    - Implication Proof 
+sometimes called as local information
+- Claim: a statement submitted to Universal Decision Contract 
+- Property : submit as a claim to Universal Decision Contract. Contains
+    - Decider: explained in the next section 
+    - Input: msg + pub key. What is being decided by the decideder specified above. 
+    - Witness aka Implication Proof: proof associated with the input
```
Comment (syuhei176) [inline articles/OVM_architecture.md]: Property don't have witness.
And there are differences between smart contract and client.
part of local information is used as Implication Proof for smart contract. And Implication Proof is data to prove implication(Property to another property).
And also local information is used for that client decide claim. Part of local information called witness in client.

Comment (syuhei176) [reply articles/OVM_architecture.md]: Screen Shot 2019-08-23 at 11 21 51
local information includes witness and property.

Comment (syuhei176) [reply articles/OVM_architecture.md]: `Foo(i) -> Foo(5)`

File: articles/OVM_architecture.md (hunk 1 above)
Comment (yuriko627) [reply articles/OVM_architecture.md]: >and name is now UniversalAdjudicator.sol
Should we also follow PG in terms of all the namings? To what extent do you think we have to share stuff with other teams for the OVM standardization?
>client can decide the claim is whether true or false by local information.
I'll review and add this part.

File: articles/OVM_architecture.md (hunk 2 above)
Comment (yuriko627) [reply articles/OVM_architecture.md]: Sorry, wrong indent. What I meant is Property and Witness are both information that decider use to make optimistic decisions.
Also, I believe there is another kind of witness which can be evidence of input like this: https://hackmd.io/@aTTDQ4GiRVyyce6trnsfpg/S1yGmXVxB#WitnessExists on the line 10

File: articles/OVM_architecture.md (hunk 1 above)
Comment (syuhei176) [reply articles/OVM_architecture.md]: Especially UniversalAdjudicator.sol should be shared implementation and name. Specific predicate can be different. There are things we should follow standard and need not to follow.

Comment (yuriko627) [reply articles/OVM_architecture.md]: Ok I'll change the names and interfaces.

File: articles/OVM_architecture.md [hunk 3]
```diff
@@ -1,70 +1,90 @@
...
+Also, I'll just leave amazing resources for those of you who want to start with general OVM introductory article here 😊:
+- [Introducing the OVM](https://medium.com/plasma-group/introducing-the-ovm-db253287af50)
+- [The OVM loves your scaling solution](https://medium.com/plasma-group/introducing-the-ovm-db253287af50)
+![](../assets/OVM_Architecture_V1.jpg)
+
+[Components - Organs] 
+- OVM: L1 contracts + Client Decider 
+    - **Universal Adjudication Contract**: core of the OVM, single theory of commanding all the L2 scaling constructions 
+    - **Deposit&Exit/Dispute Contract**: generic contract to handle deposit, standard exit request, and exit dispute process. Initial states are locked into this contract before they are updated in the L2.  
+    - **Predicate Contract**: called by Universal Adjudication Contract to validate/dispute the submitted property
```
Comment (syuhei176) [inline articles/OVM_architecture.md]: Some general predicate contract(ForAllSuchThat, Not, And, Or) and predicates(SignedByPredicate...etc) are in L1.
Deciders and quantifiers are in L2.

**Pull request merged**